sklearn/metrics/cluster/adjacency_matrix.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def adjacency_matrix(cluster_assignement):
```

Comment:
this is already implemented in ``contingency_matrix``

Reply:
Yes, these are different things.